openvpn-gui-res.h

```diff
@@ -97,6 +97,7 @@
 #define ID_RB_BALLOON1                   240
 #define ID_RB_BALLOON2                   241
 #define ID_CHK_SHOW_SCRIPT_WIN           242
+#define ID_CHK_ALWAYS_USE_ISERVICE     243
```

Comment:
Align 243 to 242 on previous line? Ignore if no critical changes are needed.

Reply:
will do!